process/templates/WG_NAME_graduation_stage.md

```diff
@@ -1,11 +1,41 @@
 ## Working Group graduation application
 
+### WG has met all Incubating requirements
+  * "link to Incubating PR"
+
+### List WG Chair(s) and or Vice Chair
+The WG must have a minimum of 2 Chair(s) or 1 Chair and 1 Vice Chair
+
+  * "name, affiliation, GitHUb ID"
+
 ### List of regular contributors
 The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
-
   * "name, affiliation, GitHub ID"
 
-### Working Group activity
-The WG must have met at least 4 times over a period of at least 2 months since becoming `Incubating`
+### Release cadence
+The TI must be able to show a consistent release cadence.
+  * "link to release documentation"
+
```

Comment:
```suggestion
```

I think this is a good idea that we should discuss further (see for example https://repos.openssf.org/), but it is not currently a requirement today.